website/docs/developer-docs/docs/style-guide.mdx

```diff
@@ -2,151 +2,216 @@
+If you notice any inconsistencies, feel free to open an [Issue](https://github.com/goauthentik/authentik/issues) or submit a [Pull Request](https://github.com/goauthentik/authentik/pulls) to address them.
 
-If you find any documentation that doesn't match these guidelines, feel free to either open an [Issue](https://github.com/goauthentik/authentik/issues) or a [PR](https://github.com/goauthentik/authentik/pulls) so they can be fixed.
+- [General Style Guidelines](#general-style-guidelines)
+- [Terminology](#terminology)
+- [Writing Style](#writing-style)
+- [Word Choices](#word-choices)
+- [Formatting Guidelines](#formatting-guidelines)
+- [Component-Based Formatting](#component-based-formatting)
+- [Error Message Formatting and Troubleshooting](#error-message-formatting-and-troubleshooting)
+- [Accessibility Best Practices](#accessibility-best-practices)
+- [React Component Imports](#react-component-imports)
+- [Notes and Warnings](#notes-and-warnings)
+
+---
 
 ## General style guidelines
 
-- Try to order the documentation sections in the order that makes it easiest for the user to follow. That is, order the sections in the same order as the actual workflow used to accomplish the task.
+- **Logical Order:** Documentation should be structured to follow the natural order of tasks, making it easier for users to follow. Organize sections in a manner that reflects the actual workflow used to complete tasks.
+- **Headings:** Use headings (sub-titles) to break up large blocks of text, making it easier for users to navigate the content and find specific sections quickly.
+- **Cross-References:** Always include cross-references to related content. If a concept is referenced elsewhere in the documentation, link to the relevant section to provide users with additional context or instructions.
+- **Relative Paths:** Use relative paths when linking to documentation files. This will ensure links are automatically updated if file paths change in the future.
+- **Markdown File Type:** The standard file type for documentation is `.md`. Use `.mdx` only if React components, such as interactive elements, are required.
 
-- Use headings (sub-titles) to break up long documents, and make it easier to find a specific section.
+## Terminology
 
-- Add cross-reference links to related content whenever possible.
+### authentik product naming conventions
 
-- You can use standard [Docusaurus-specific features](https://docusaurus.io/docs/next/markdown-features), which include MDX elements such as tabs and admonitions.
+- The product name **authentik** should always be written with a lowercase "a" and a "k" at the end, even if it begins a sentence. This consistent style should be followed throughout the documentation.
+- The company name is **Authentik Security, Inc.**, but for non-legal documentation, you may shorten it to **Authentik Security**.
 
-## Terminology
+### Industry terms and technology names
```

Comment:
We need to keep the heading of ## Terminology. That is a standard and expected section for any Style Guide,